README.md

```diff
@@ -1,18 +1,25 @@
 # Trial task for back-end developers
 
-Please use Node.js 12+, MySQL and a web framework of your choice to implement the following trial task. You can use TypeScript and any node libraries you think are necessary or helpful for you, but it helps us a lot if you keep it minimal. It is recommended to use some code-formatter and linter of your choice (we recommend [Prettier](https://prettier.io/) and [XO](https://github.com/xojs/xo)).
+Someone from support notified us that users are using too many times passwords like "password", "dog123", or "heeello" and their accounts are getting hacked by their neighbours to see private pictures of their private parts.
+We need this fixed ASAP, but our developers are busy working on the next-cool-feature™ using the next-cool-stack™, and they have no time for implementing the changes needed to fix the issue. So our only chance is YOU!*
```

Comment:
Note that in most flavours of Markdown, a newline does not mean a new paragraph. (You need two newlines to mark a new paragraph).

Don't know if you mean this as a new paragraph or not.

Reply:
No, it's not meant to be a new paragraph as it's a continuation of the topic before. I broke it into a new line for ease of editing.